src/core/qgspallabeling.cpp

```diff
@@ -61,6 +61,25 @@
 #include "qgscurvepolygon.h"
 #include <QMessageBox>
 
+// TODO: Move to qgis.h?
```

Comment:
@3nids is this method already available somewhere? Or any input where to move it?

Reply:
Indeed it's already available, I think in qgis.h.